File: 백준/Bronze/23627. driip/driip.c
#include <stdio.h>
#include <string.h>

int main(void){
  char S[1000];
  scanf("%s", &S);
  int len = strlen(S);
  if (len >= 5 && strcmp(S+len-5, "driip")==0){
    printf("cute\n");
  } else {
    printf("not cute\n");
  }

  return 0;
}

File: 백준/Bronze/23627. driip/driip.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String S = sc.nextLine();
        if (S.length() >= 5 && S.endsWith("driip")){
            System.out.println("cute");
        } else {
            System.out.println("not cute");
        }
    }
}